### PR TITLE
Add `hardwareId` support

### PR DESCRIPTION
**Pull request by cmaglie**
See https://github.com/arduino/arduino-cli/issues/2024 for more info.

### CODE REVIEW

**Review by per1234 — CHANGES_REQUESTED** [review]
**UPDATE**: Resolved by https://github.com/arduino/serial-discovery/pull/60/commits/83aaf3048f11f67722072d5ecdd74b16146a635e

<s>
This outdated statement in the readme should be updated to accompany the bump to using Go 1.19 for development and maintenance of the project:

https://github.com/arduino/serial-discovery/blob/fa69a61965ca8128d9a9403cc9a476186266c7a9/README.md?plain=1#L7
</s>

File: DistTasks.yml [hunk 1]
```diff
@@ -184,7 +184,7 @@ tasks:
       #
       # Until there is a fix released we must use a recent gcc for Linux_ARMv6 build, so for this
       # build we select the debian10 based container.
-      CONTAINER_TAG: "{{.GO_VERSION}}-armel-debian10"
+      CONTAINER_TAG: "{{.GO_VERSION}}-armel-debian9"
```
Comment (umbynos) [inline DistTasks.yml:187]: why this was necessary?

Comment (cmaglie) [reply DistTasks.yml:187]: I copied it from the [`DistTask.yml`](https://github.com/arduino/arduino-cli/blob/master/DistTasks.yml) in the Arduino CLI repo.

I guess that @MatteoPologruto tried to go back to debian9 to see if the problem in the comment was fixed (and it seems fixed indeed) but didn't update the comment.
We should:
* Update the comment (remove it)
* Update the template in the asset repo

**Comment by cmaglie** [comment]
> This outdated statement in the readme should be updated to accompany the bump to using Go 1.19

I've just removed the reference to the go version since it's now enforced by `go.mod`.

**Review by umbynos — APPROVED** [review]

**Pull request merged**